### PR TITLE
build and download OneAuthTestApp

### PR DESCRIPTION
**Pull request by cikelengfeng**
Ensure the existing pipelines can consume new version of Common SDK and publish new build of OneAuthTestApp to Broker’s particular feed. 

OneAuth: [Pipelines - Runs for Android (visualstudio.com)](https://office.visualstudio.com/OneAuth/_build?definitionId=6332&_a=summary) 

Change the “auth-client-android-dev” pipeline to trigger the above pipeline and pass the version of Common SDK to it. 

Change the “Android Broker CI” pipeline to be able to download the new apk of OneAuthTestApp and . 

msal_with_broker_high_api in the broker-test.yml file 

msal_with_brokerhost_high_api in the broker-test.yml file 

msal_with_broker_low_api in the broker-test.yml file

### CODE REVIEW

File: azure-pipelines/ui-automation/broker-test.yml [hunk 1]
```diff
@@ -197,7 +201,7 @@ stages:
           displayName: 'Publish Msal Test apk for later use'
           artifact: msalE2ETestApp
         - task: UniversalPackages@0
-          displayName: 'Publish Msal E2E Test apk to internal feed'
+          displayName: 'Push Msal E2E Test apk to internal feed'
```
Comment (somalaya) [inline azure-pipelines/ui-automation/broker-test.yml:204]: is this change needed?

File: azure-pipelines/ui-automation/broker-test.yml [hunk 2]
```diff
@@ -206,7 +210,58 @@ stages:
...
+        - task: PowerShell@2
+          displayName: Queue and wait for OneAuth Test App Apk generation pipeline
+          name: buildApk
+          continueOnError: true
+          inputs:
+            filePath: '$(Build.SourcesDirectory)/azure-pipelines/scripts/queue-build.ps1'
+            arguments: '-OrganizationUrl "https://office.visualstudio.com/" -Project "OneAuth" -PipelinePAT "$(OfficePAT)" -WaitTimeoutInMinutes 120 -BuildDefinitionId "$(oneAuthTestAppPipelineId)"'
+            workingDirectory: '$(Build.SourcesDirectory)'
+        - task: DownloadExternalBuildArtifacts@15
+          displayName: 'Download New Built OneAuth test app'
```
Comment (somalaya) [inline azure-pipelines/ui-automation/broker-test.yml:233]: How about simply "Download OneAuthTestApp"? Because we are not building OneAUthTestAPp in our pipeline

**Review by somalaya — APPROVED** [review]
LGTM

File: azure-pipelines/ui-automation/broker-test.yml (hunk 1 above)
Comment (cikelengfeng) [reply azure-pipelines/ui-automation/broker-test.yml:204]: Yes, the word "publish" might cause confusion with Azure's publish task (the above 'Publish Msal Test apk for later use' task is a publish task).

File: azure-pipelines/ui-automation/broker-test.yml (hunk 2 above)
Comment (cikelengfeng) [reply azure-pipelines/ui-automation/broker-test.yml]: Sure, changed.

**Comment by fadidurah** [comment]
Have we validated these new pipeline steps? Like make sure that nothing is breaking with these additions?

**Comment by cikelengfeng** [comment]
> Have we validated these new pipeline steps? Like make sure that nothing is breaking with these additions?

Yes, I ran the Android Broker CI pipeline entirely, I saw there were some test cases failed but It shouldn't be caused by this PR.

**Review by fadidurah — APPROVED** [review]
